prow/cmd/pipeline/controller.go

```diff
@@ -300,6 +305,15 @@ func (c *controller) patchProwJob(pj *prowjobv1.ProwJob, newpj *prowjobv1.ProwJo
 	return pjutil.PatchProwjob(context.TODO(), c.pjc.ProwV1().ProwJobs(c.pjNamespace()), logrus.NewEntry(logrus.StandardLogger()), *pj, *newpj)
 }
 
+func (c *controller) listProwJobs(namespace string) ([]*prowjobv1.ProwJob, error) {
+	pjnl := c.pjLister.ProwJobs(namespace)
+	pjs, err := pjnl.List(labels.NewSelector())
+	if err != nil {
+		return nil, err
+	}
+	return pjs, nil
```

Comment:
I'd say you could simply `return pjnl.List(labels.NewSelector())` here, the caller is still supposed to test `err` and avoid using the returned value /shrug

Reply:
good catch!